### DISCUSSION TITLE
How can I get the public sponsor badge?

### DISCUSSION THREAD

**Discussion by PT-3Y** [community/community#63395] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I want to get sponsor badge.
But I am not sure how can I get it.
If anyone knows about it, let me know.

**Comment by workwithhim** [comment, marked as answer]
If you want to get the public sponsor badge on github profile, you can check the following link:
https://github.com/sponsors/caioagiani

**Reply by caioagiani** [reply]
up!

**Comment by PT-3Y** [comment]
@workwithhim
Thanks for your quick answer.
